content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -143,7 +143,6 @@ different Kubernetes components.
 | `ServerSideApply` | `false` | Alpha | 1.14 | 1.15 |
 | `ServerSideApply` | `true` | Beta | 1.16 | |
 | `ServiceAccountIssuerDiscovery` | `false` | Alpha | 1.18 | |
-| `ServiceAppProtocol` | `false` | Alpha | 1.18 | |
```

Comment:
Looks good.
Could you pls move line 141/142 here so that the alphabet order is preserved?